### DISCUSSION TITLE
如何指定扫码登录

### DISCUSSION THREAD

**Discussion by LittleChest** [DreamVoid/MiraiMC#425] [问答 / Q&A] [answered]
我在文档中没有找到如何指定扫码登录，如果不指定密码MiraiMC就会报错

**Comment by ApliNi** [comment, marked as answer]
https://github.com/DreamVoid/MiraiMC/releases/tag/v1.8-pre2

---

## 新增
添加getNameCard获取群名片方法
添加扫码登录和自动打开二维码文件功能

> 用法：`/mirai login 机器人QQ QRCode ANDROID_WATCH/MACOS`
> 即密码部分填写“QRCode”，协议填写“ANDROID_WATCH”或“MACOS”即可使用扫码登录。
> 在 Windows 下还支持调用系统图片查看器打开二维码图片，只需将 general.auto-open-qrcode-file 设为 true，但请注意恶意程序可能会利用此功能执行任意代码，原理是 MiraiMC 直接运行命令行 explorer 图片路径，因此请自行斟酌安全性。
> mirai 仍在测试扫码登录功能，MiraiMC 对此的支持可能会随时变化或失效。

**Reply by LittleChest** [reply]
文档没翻到这段 ;w;

**Reply by ApliNi** [reply]
可能因为这是预发布吧
